src/rt/debug/harness.h

```diff
@@ -155,7 +157,11 @@ class SystematicTestHarness
 #else
       UNUSED(seed);
 #endif
-      sched.init(cores);
+
+      if (run_at_termination)
+        sched.init(cores, run_at_termination);
+      else
+        sched.init(cores);
```

Comment:
```suggestion

      sched.init(cores, run_at_termination);
```
I think this is the same.